Coverity scan failed

**Describe the bug**

https://github.com/longturn/freeciv21/actions/runs/1908529390
```
CMake Error at /usr/lib/x86_64-linux-gnu/cmake/Qt5/Qt5Config.cmake:28 (find_package):
  Could not find a package configuration file provided by "Qt5Svg" with any
  of the following names:

    Qt5SvgConfig.cmake
    qt5svg-config.cmake
```

**Additional context**
New dependency

--- a/.github/workflows/build.yaml
+++ b/.github/workflows/build.yaml
@@ -42,7 +42,6 @@ jobs:
             gettext \
             qt5-default \
             libqt5svg5-dev \
-            libqt5svg5-dev \
             libkf5archive-dev \
             liblua5.3-dev \
             libsqlite3-dev \

--- a/.github/workflows/coverity.yaml
+++ b/.github/workflows/coverity.yaml
@@ -23,6 +23,7 @@ jobs:
             python3 \
             gettext \
             qt5-default \
+            libqt5svg5-dev \
             libkf5archive-dev \
             liblua5.3-dev \
             libsqlite3-dev \

